src/components/ProgressIndicator/CompletionMark.js

```diff
@@ -1,11 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import {useIntl} from 'react-intl';
 
 import FAIcon from 'components/FAIcon';
 
 const CompletionMark = ({completed = false}) => {
+  const intl = useIntl();
+  // Wrapper may be a DOM element, which can't handle <FormattedMessage />
+  const ariaLabel = intl.formatMessage({
+    description: 'Completion message',
```

Comment:
Try to be as specific as possible in these descriptions - they are used to generate the ID of a message.

```suggestion
    description: 'Step completion marker icon label',
```